NumLib/DOF/LocalToGlobalIndexMap.cpp

```diff
@@ -94,7 +94,13 @@ void LocalToGlobalIndexMap::findGlobalIndices(
                 continue;
             MeshLib::Location l(
                 mesh_id, MeshLib::MeshItemType::Node, (*n)->getID());
-            indices.push_back(_mesh_component_map.getGlobalIndex(l, comp_id));
+            auto const global_index =
+                _mesh_component_map.getGlobalIndex(l, comp_id);
+            if (global_index == std::numeric_limits<GlobalIndexType>::max())
```

Comment:
Just out of curiosity: In which cases this will happen?

Reply:
The mesh component map returns -1 if there is no DOF associated with the given location and the component. In case of a variable (like pressure) defined only in the base nodes and given quadratic mesh elements, the higher order node locations don't have that variable associated there.